src/plugin/file/kml/ui/kmlexportui.js

```diff
@@ -17,7 +17,8 @@ plugin.file.kml.ui.kmlExportDirective = function() {
     restrict: 'E',
     scope: {
       'exporter': '=',
-      'simple': '=?'
+      'simple': '=?',
+      'savedplaces': '='
```

Comment:
Avoid referring to Saved Places anywhere outside the places plugin (`src/plugin/places`). I believe the `simple` attribute is already what you want, but it should also disable the `Use Feature Colors` option.